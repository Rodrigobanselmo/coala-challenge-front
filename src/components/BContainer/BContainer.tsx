import React, { FC, ReactNode } from "react";

import { Box } from "@mui/material";

import { SContainerProps } from "./types";

export const BContainer: FC<{ children: ReactNode } & SContainerProps> = ({
  children,
  ...props
}) => {
  return (
    <Box px={[10, 10, 15]} py={[10, 10, 15]} {...props}>
      {children}
    </Box>
  );
};
