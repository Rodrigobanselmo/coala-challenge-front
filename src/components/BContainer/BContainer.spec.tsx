import React from "react";
import { render } from "@testing-library/react";
import { BContainer } from "./BContainer";

describe("BContainer", () => {
  it("matches the snapshot", () => {
    const { container } = render(
      <BContainer>
        <div>Content goes here</div>
      </BContainer>
    );
    expect(container).toMatchSnapshot();
  });
});
