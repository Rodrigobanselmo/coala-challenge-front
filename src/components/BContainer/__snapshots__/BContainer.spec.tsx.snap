// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BContainer matches the snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-t0di4r"
  >
    <div>
      Content goes here
    </div>
  </div>
</div>
`;
